src/concepts/html/README.md

```diff
@@ -2,17 +2,28 @@
 description: The procedural macro for generating HTML and SVG
 ---
 
-# Using html!
+# Using [`html!`][yew_macro_html]
 
-The `html!` macro allows you to write HTML and SVG code declaratively. It is similar to JSX \(a Javascript extension which allows you to write HTML code inside of Javascript\).
+The [`html!`][1] macro allows you to write HTML and SVG code declaratively. It is similar to JSX \(a Javascript extension which allows you to write HTML code inside of Javascript\).
 
 **Important notes**
 
-1. The `html!` macro only accepts one root html node \(you can counteract this by [using fragments or iterators](lists.md)\)
-2. An empty `html! {}` invocation is valid and will not render anything
-3. Literals must always be quoted and wrapped in braces: `html! { "Hello, World" }`
+
+- The [`html!`][yew_macro_html] macro only accepts one root HTML node (you can overcome this by [using fragments or iterators](lists.md))
+- An empty `html! {}` invocation is valid and will not render anything.
+- Literals inside of tags must always be quoted and wrapped with braces (this is different to attribute values - see below)
+  * `html! { "Hello, World" }`
+  * `html! { <div>{ "Hell, World" }</div> }`
+  * `html! { <div>{ String::from("foo") + "bar" }</div>`
+- Quoted attribute values are taken literally. The value is set at compile-time and does not change at run-time.
+  * `html! { <div> id="bar"</div> }`
+- Unquoted attribute values are interpreted as expressions and therefore have to be valid Rust expressions.
+  * `let foo = "bar"; html! { <div id=foo></div> }`
+  * `html! { <div id=String::from("foo") + "bar"></div> }`
```

Comment:
This is a bit confusing because literals are also expressions. I'd prefer if we left these notes out